test/checkoutput/README.md

```diff
@@ -0,0 +1,54 @@
+# checkoutput tool
+
+The checkoutput tool parses a text or markdown that has a command and the expected output sequence, runs the command and checks the output of the command against the expected output. 
+
+**Usage:**
+```
+<tool> -parsefile <path/to/file> -verbose
+
+-verbose is optional
+-parsefile should specify the path of the input markdown file.
+```
+
+The tool has 2 major functions
+
+1. Parsing and capturing the Run/Verify blocks
+2. Run and verify the output of the blocks captured in the previous step
+
+## Parsing
```

Comment:
This belongs into source code comments, not the end-user documentation (= README.md). What we need here is a definition of how a test case gets defined and in which environment it gets executed.

For example, what is the current directory? I think it should be the directory that is being tested. That's consistent with "go test" and enables referencing local files without a long path that might change when code gets moved around.

Also, `go run ./staging/....` doesn't work in k/k. One has to be inside the module defined inside `staging`.

Reply:
I will move the parsing part into the go file. We will move to a directory structure instead of specify files, and that should serve as the current directory. I will change the code to reflect it.